platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -87,7 +95,7 @@ public static String sanitizeFilename(String filename) {
      * @param mimetype
      * @return true if the mime type is acceptable
      */
-    public static boolean checkForClientSideVulnerableMimeType(String mimetype) {
+    public static boolean isMimeTypeClientSideVulnerable(String mimetype) {
```

Comment:
@coyotesqrl I suggested this name change to capture the spirit of using forms of "to be" for boolean-returning methods, but totally agree that the semantics of it were reversed. Good catch. We could always rename it to something like `isMimeTypeClientSideNotVulnerable` to really confuse people :)